Doc/whatsnew/3.10.rst

```diff
@@ -556,10 +556,10 @@ Optimizations
   (Contributed by Victor Stinner in :issue:`41006`.)
 
 * The ``LOAD_ATTR`` instruction now uses new "per opcode cache" mechanism.  It
-  is about 36% faster now. This makes optimized ``LOAD_ATTR`` instructions the
-  current most performance attribute access method (faster than slots).
-  (Contributed by Pablo Galindo and Yury Selivanov in :issue:`42093`, based on
-  ideas implemented originally in PyPy and MicroPython.)
+  is about 36% faster now for regular attributes and 30% for slots.
```

Comment:
Honestly the 30% is a pretty wild guess -- your 36% clearly is not. I'm not even sure how to really measure this, given that when I build Python from source it's somewhat slower than the python.org binary. I assume this is because of PGO or other compiler flags. (Not `--with-pydebug`, that is much slower still.)

Up to you if you still want to put it this way.

Reply:
I did some more rigurous measurements for __slots__ and I have updated the text.